src/enums/cube-type.ts

```diff
@@ -0,0 +1,6 @@
+export enum CubeType {
+  DuckDBProtoCube = 'duckdb_proto_cube',
+  DuckDBCube = 'duckdb_cube',
+  PostgresProtoCube = 'postgres_proto_cube',
+  PostgresCube = 'postgres_cube'
```

Comment:
Will the duckdb cubes eventually go away?

Reply:
They should... Right now they haven't... Also we need to produce the cube file for Download.  Although the code is moving in to place that there should be a lot less DuckDB usage.